apps/mull-ui/src/app/pages/create-event/create-event.scss

```diff
@@ -0,0 +1,80 @@
+@import 'variables';
+
+.create-event {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 95%;
+  margin: auto;
+  .custom-text-input-container {
+    margin-bottom: 1.875rem;
+  }
+
+  .custom-time-picker-container {
+    margin: 0 0 1.25rem;
+  }
+}
+
+.event-image-upload {
+  display: none;
+}
+
+.event-input-border {
+  border-radius: 8px;
+  border: 1.5px $primary-color solid;
+  &.error {
+    border-color: #ff0000;
+  }
```

Comment:
See `libs/ui-lib/src/lib/custom-text-input/custom-text-input.scss`. This style block is a duplicate and can be moved to a mixin or separate class in ui-lib.